libsolidity/ast/TypeProvider.cpp

```diff
@@ -225,16 +225,20 @@ Type const* TypeProvider::fromElementaryTypeName(ElementaryTypeNameToken const&
 	case Token::BytesM:
 		return fixedBytes(m);
 	case Token::FixedMxN:
+		solUnimplemented("Not yet Implemented - FixedPointType.");
```

Comment:
```suggestion
		solUnimplemented("Not yet implemented - FixedPointType.");
```

Reply:
iiuc, this may reduce the diff a bit more (and be consistent with previous error message).

Reply:
Fixed!